src/args.jl

```diff
@@ -1562,3 +1555,19 @@ function _add_defaults!(d::KW, plt::Plot, sp::Subplot, commandIndex::Int)
     _replace_linewidth(d)
     d
 end
+
+function _series_index(d, sp)
+    idx = 0
+    for series in series_list(sp)
+        if series[:primary]
+            idx += 1
+        end
+        if series == d
```

Comment:
Not entirely sure what this does?

Reply:
This determines the "primary index" of the series in the subplot.
A short explanation, how it ended up here:
This piece of code was actually part of `_add_defaults!` and at first I did not plan to remove the series processing of this function, but I also needed the series index in `_update_series_attributes!`. So I put this code in a separate function `series_index`, that would be called by both `_add_defaults!` and `_update_series_attributes!`. It turned out that series processing in `_add_defaults!` was no longer necessary, so I removed all that. ... Hence, the separate `series_index` function.

Reply:
Missing the :mind_blown:  reaction